packages/maskbook/src/plugins/Snapshot/UI/InformationCard.tsx

```diff
@@ -0,0 +1,132 @@
+import { useContext } from 'react'
+import { Box, Link, createStyles, makeStyles, Typography, Avatar } from '@material-ui/core'
+import { format } from 'date-fns'
+import OpenInNew from '@material-ui/icons/OpenInNew'
+
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { SnapshotCard } from './SnapshotCard'
+import { TokenIcon } from '../../../extension/options-page/DashboardComponents/TokenIcon'
+import { EthereumBlockie } from '../../../web3/UI/EthereumBlockie'
+import { resolveAddressLinkOnEtherscan, resolveBlockLinkOnEtherscan, resolveIPFSLink } from '../../../web3/pipes'
+import { formatEthereumAddress } from '../../Wallet/formatter'
+import { useChainId } from '../../../web3/hooks/useChainState'
+
+import { SnapshotContext } from '../context'
+import { useProposal } from '../hooks/useProposal'
+
+export interface InformationCardProps {}
+
+export interface InfoFieldProps {
+    title: string
+    children: React.ReactNode
+}
+
+const useStyles = makeStyles((theme) => {
+    return createStyles({
+        field: {
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            margin: `${theme.spacing(1)} auto`,
+        },
+        link: {
+            display: 'flex',
+            color: 'inherit',
+            alignItems: 'center',
+            marginLeft: theme.spacing(1),
+            textDecoration: 'none !important',
+        },
+        avatar: {
+            width: 16,
+            height: 16,
+        },
+        avatarWrapper: {
+            marginRight: 8,
+        },
+    })
+})
+
+export function InfoField(props: InfoFieldProps) {
+    const classes = useStyles()
+
+    return (
+        <Box className={classes.field}>
```

Comment:
If you're not using Box functionality, I don't think it should be a Box. Maybe a div will be better